test/Microsoft.TestPlatform.PerformanceTests/TranslationLayer/EventHandler/RunEventHandler.cs

```diff
@@ -57,6 +51,7 @@ public void HandleLogMessage(TestMessageLevel level, string message)
         }
 
         Metrics = testRunCompleteArgs.Metrics;
+        LogMessages.Add($"[ERROR] {testRunCompleteArgs.Error}");
```

Comment:
Is it always an error case to come here?

Reply:
nope added condition.